pkg/installer/custominstallconfig_test.go

```diff
@@ -363,3 +374,54 @@ func verifyIgnitionFiles(t *testing.T, temp map[string]any, storageFiles []strin
 	}
 	assert.EqualValues(t, "ARO", config.Data["invoker"])
 }
+
+func verifyMasterPointerIgnition(t *testing.T, ignData []byte) {
+	ignContents := &igntypes.Config{}
+	err := json.Unmarshal(ignData, &ignContents)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	actualSource := *ignContents.Ignition.Config.Merge[0].Source
+	assert.EqualValues(t, expectedMasterIgnitionSource, actualSource, fmt.Sprintf("expected master pointer ignition to be %s but found %s", expectedMasterIgnitionSource, actualSource))
+}
+
+func verifyUpdateMCSCertKey(t *testing.T, bootstrap *bootstrap.Bootstrap) {
+	config := &igntypes.Config{}
+	config = bootstrap.Config
+
+	cert := &x509.Certificate{}
+	var data []byte
+
+	for i, fileData := range config.Storage.Files {
+		if fileData.Path == mcsCertFile {
+			contents := config.Storage.Files[i].Contents.Source
+			replaceable := "data:text/plain;charset=utf-8;base64,"
```

Comment:
nit: Just need to split by comma and take the latter part of the string. Not really an issue but this header "might" change in the future.

Reply:
@rna-afk I have been able to eliminate this in the decode logic as pointed above. At this time, there still seems to be a need for `header` while updating cert and key files. I'll look into how that can be eliminated too.